### PR TITLE
 [finishes #163047049] created tests for meetups endpoints

### PR DESCRIPTION
**Pull request by samwanjo41**
**What does this PR do?**
Tests for the meetups endpoints

**Description of task to be completed?**
Write unittests for meetups endpoints

**How should you manually test this?**
Clone the repo git clone https://github.com/samwanjo41/Questioner.git
cd into the project folder
Activate virtual env: source/bin/activate
Use POSTMAN to test the API endpoints
Navigate to app/tests/v1
On the terminal, run pytest

**What are the relevant PT stories?**
#163047049

**Screenshots**
https://open-andela.slack.com/files/U6U7ENVE1/FFCRKCWTT/image.png
image

### CODE REVIEW

File: app/tests/v1/test_meetups.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            "tags": ["python", "Ihub"]
+        }
+
+       
+
+      
+    def test_upcoming_meetups(self):
+        """user view upcoming meetups"""
+        response = self.client.get('/api/v1/meetups/upcoming', data=json.dumps(self.record1), content_type='application/json')
+        self.assertEqual(response.status_code, 200)
```
Comment (dessHub) [inline app/tests/v1/test_meetups.py:41]: Also, assert the `message` or `data` not just `status code.`

File: app/tests/v1/test_meetups.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    def test_specific_meetups(self):
+        """user view specific meetup by id"""
+        response = self.client.get('/api/v1/meetups/1', data=json.dumps(self.meetup), content_type='application/json')
+        self.assertEqual(response.status_code, 200)
```
Comment (dessHub) [inline app/tests/v1/test_meetups.py:46]: Same here. Add `message` or `data ` assertions.

**Comment by dessHub** [comment]
Fix the failing tests.

File: app/tests/v1/test_meetups.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            "happeningOn": "happeningOn",
+            "tags": ["python", "Ihub"]
+        }
+
+       
+
+      
+    def test_upcoming_meetups(self):
+        """user view upcoming meetups"""
+        response = self.client.get('/api/v1/meetups/upcoming', data=json.dumps(self.record1), content_type='application/json')
```
Comment (dessHub) [inline app/tests/v1/test_meetups.py:40]: Why are you dumping data on a `GET method?

File: app/tests/v1/test_meetups.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        self.assertEqual(response.status_code, 200)
+
+    def test_specific_meetups(self):
+        """user view specific meetup by id"""
+        response = self.client.get('/api/v1/meetups/1', data=json.dumps(self.meetup), content_type='application/json')
```
Comment (dessHub) [inline app/tests/v1/test_meetups.py:45]: Still dumping data on a `GET` method.

File: app/tests/v1/test_meetups.py (hunk 2 above)
Comment (samwanjo41) [reply app/tests/v1/test_meetups.py:46]: okay got it

**Pull request merged**